backend/manager/modules/bll/src/main/java/org/ovirt/engine/core/bll/RunVmCommand.java

```diff
@@ -385,6 +386,19 @@ private void releaseHostDevicesLock() {
         }
     }
 
+    private void warnIfRequiredHostDeviceUsed() {
+        if (!needsHostDevices) {
+            return;
+        }
+        Guid hostId = getVm().getDedicatedVmForVdsList().get(0);
```

Comment:
What will happen if the VM is pinned to multiple hosts?

Reply:
@smelamud how about copying the comment we have in other places that do this (`// Only single dedicated host allowed for host devices, verified on validates`)?